examples/rdkafka_simple_producer.c

```diff
@@ -83,28 +90,68 @@ static void dr_msg_cb (rd_kafka_t *rk,
 int main (int argc, char **argv) {
         rd_kafka_t *rk;         /* Producer instance handle */
         rd_kafka_topic_t *rkt;  /* Topic object */
+        rd_kafka_topic_conf_t * tconf; /* Topic conf */
         rd_kafka_conf_t *conf;  /* Temporary configuration object */
         char errstr[512];       /* librdkafka API error reporting buffer */
         char buf[512];          /* Message value temporary buffer */
-        const char *brokers;    /* Argument: broker list */
-        const char *topic;      /* Argument: topic to produce to */
-
-        /*
-         * Argument validation
-         */
-        if (argc != 3) {
-                fprintf(stderr, "%% Usage: %s <broker> <topic>\n", argv[0]);
-                return 1;
+        const char *brokers = NULL;    /* Argument: broker list */
```

Comment:
The rdkafka_simple_producer was added because rdkafka_example.c was getting heavy with options trying to show-case everything.
I'd like the simple producer to stay simple.

We can wait with example code until the config-string-based partitioner support is in place (so you could do `conf_set("partitioner", "murmur2", ..)`).

So please remove the patch for this file.